docs/source/processes.rst

```diff
@@ -424,79 +428,185 @@ This section will first describe the basics of this request format, and after go
+stored locally and exposed externally with returned reference URL. The other mode ``value`` returns the contents
+directly in the response instead of the URL.
 
-.. note::
-    Other parameters can be added to the request to provide further functionalities. Above fields are the minimum
-    requirements to request a :term:`Job`. Please refer to the |exec-api|_ definition for all applicable features.
+When ``outputs`` section is omitted, it simply means that the :term:`Process` to be executed should return all
+outputs it offers in the created :ref:`Job Results <proc_op_result>`. In such case, because no representation modes
+is specified for individual outputs, `Weaver` automatically selects ``reference`` as it makes all outputs more easily
+accessible with distinct URL afterwards. If the ``outputs`` section is specified, but that one of the outputs defined
+in the :ref:`Process Description <proc_op_describe>` is not specified, that output should be omitted from the produced
+results. For the time being, because only ``reference`` representation is offered for produced output files, this
+filtering is not implemented as it offers no additional advantage for files accessed directly with their distinct URLs.
+This could be added later if ``Multipart`` raw data representation is required.
+Please |submit-issue|_ to request this feature if it is relevant for your use-cases.
 
-.. note::
-    Since most of the time, returned files are not human readable or are simply too large to be displayed, the
-    ``transmissionMode: value`` is rarely employed. Also, it is to be noted that outputs representing ``LiteralData``
-    (which is even more uncommon) would automatically be represented as ``value`` without explicitly requesting it,
-    as there would not be any file to return. If this poses problem or you encounter a valid use-case where ``value``
-    would be useful for your needs, please |submit-issue|_ to request the feature.
+.. fixme::
+    Filtering of ``outputs`` not implemented (everything always available).
+    https://github.com/crim-ca/weaver/issues/380
 
-.. |exec-api| replace:: OpenAPI Execute
-.. _exec-api: `exec-req`_
+Other parameters presented in the above examples, namely ``mode`` and ``response`` are further detailed in
+the following :ref:`proc_exec_mode` section.
+
+.. _proc_exec_mode:
+
+Execution Mode
+~~~~~~~~~~~~~~~~~~~~~
+
+In order to select how to execute a :term:`Process`, either `synchronously` or `asynchronously`, the ``Prefer`` header
+should be specified. If omitted, `Weaver` defaults to `asynchronous` execution. To execute `asynchronously` explicitly,
+``Prefer: respond-async`` should be used. Otherwise, the `synchronous` execution can be requested
+with ``Prefer: wait=X`` where ``X`` is the duration in seconds to wait for a response. If no worker becomes available
+within that time, or if this value is greater than ``weaver.exec_sync_max_wait``, the :term:`Job` will resume
```

Comment:
If `weaver.exec_sync_max_wait` is 30 seconds and I submit a job using `Prefer: wait=60`, does the job resumes asynchronously directly or after 30 seconds?
Is this mode switching covered in the spec or is it a thing specific to Weaver? I feel like if I want a sync job to complete in 30 seconds I'm not necessarily ready to handle an async job...

Reply:
Yes, it will switch automatically to async after 30sec.
The OGC spec is not very explicit about this, but they do say that the client must be ready to receive either sync/async regardless of what was requested. They mostly refer to `Prefer` header spec itself about its definition.
This parameter is defined as a *preference* that the service can decide to honour or not. Weaver responds with `Preference-Applied` header (as per the spec) to indicate what it choose to do.